src/middlewares/generateDocs.ts

```diff
@@ -0,0 +1,26 @@
+import path from 'path'
+import { logInfoMessage, logLineWithIcon, logError } from '../lib/logger.js'
+import Generator from '@asyncapi/generator'
+
+export default async (spec, config, resDir) => {
+  logInfoMessage(`Generating docs for your parsed specification...`)
+  const configData = config.generator
+  const resolvedData = spec._json
+  const generator = new Generator(
+    configData && configData.template
+      ? `@asyncapi/${configData.template}`
+      : '@asyncapi/markdown-template',
+    path.resolve(
+      resDir ? resDir : './',
+      configData && configData.folder ? configData.folder : 'docs'
+    )
+  )
+  try {
+    await generator.generateFromString(JSON.stringify(resolvedData))
+    logLineWithIcon(":zap:", "Successfully generated docs")
```

Comment:
I know you're excited about it but a zap might be too much for generated docs 😄 Especially, because we're also using it for when the service connects to the broker.

```suggestion
    logInfoMessage('Successfully generated docs')
```